1. CWE Base Mod/Cold War Era Mod (CWE)/common/scripted_effects/00_building_setup.txt

```diff
@@ -668,34 +668,49 @@ effect_starting_buildings_3 = {
 
 	effect_starting_buildings_2 = yes
 
+	## To balance the large potential deficit
+	if = {
+		limit = {
+			has_potential_resource = bg_iron_mining
+		}
+		create_building = {
+			building = "building_iron_mine"
+			level = 40
```

Comment:
I think it might be good to increase output of iron from the mine instead of massively increasing building levels

Reply:
Commit 7e25387 solves this.